src/ray/core_worker/fiber.h

```diff
@@ -140,6 +140,8 @@ class FiberState {
         std::this_thread::sleep_for(std::chrono::hours(1));
       }
     });
+
+    fiber_runner_thread_.detach();
```

Comment:
can we just remove this member variable and make it a local object just in ctor?